Add a note on inviting users (#2454)
buildkite/docs: pages/pipelines/create_your_own.md

We recommend you continue by:

> - Inviting your team to see your build and try Buildkite themselves. Invite users from your [organization's user settings](https://buildkite.com/organizations/-/users/new) by pasting their email addresses into the form. Please note that to be able to start inviting other users to your Buildkite organization, your email needs to be verified. To verify your email, go to https://buildkite.com/user/emails and click the "Resend Verification" button. Once you've verified your account, you should be able to invite users to your Buildkite organization with no issues.

Reviewer: General tightening:

```suggestion
- Inviting your team to see your build and try Buildkite themselves. Invite users from your [organization's user settings](https://buildkite.com/organizations/-/users/new) by pasting their email addresses into the form. Please note that to be able to start inviting other users to your Buildkite organization, your email needs to be verified. To verify your email, go to your [personal email settings](https://buildkite.com/user/emails) and select _Resend Verification_.
```
